verification/src/block_verifier.rs

```diff
@@ -99,6 +90,16 @@ impl<CP: ChainProvider + Clone> CellbaseVerifier<CP> {
         {
             return Err(Error::Cellbase(CellbaseError::InvalidInput));
         }
+
+        // currently, we enforce`type` field of a cellbase output cell must be absent
```

Comment:
why do we enforce this?

Reply:
Currently, we have an involuntary issue that CKB-VM execution skipped `cellbase`. It's a temporary solution until we implement `cellbase` type script verifiction.

Reply:
Feels like something we should fix before testnet.